gamemodes/terrortown/gamemode/client/cl_hud_manager.lua

```diff
@@ -118,7 +118,9 @@ function GM:HUDPaint()
 		hook.Call("HUDDrawTargetID", GAMEMODE)
 	end
 
-	HUDManager.DrawHUD()
+	if hook.Call("HUDShouldDraw", GAMEMODE, "TTTHUDManager") then
```

Comment:
I like this addition. However it should be called `TTT2HUDManger`, shouldn't it?

Reply:
it shouldn't, TTTHUDManager is just an indicator. Otherwise, we have to rename any HUDELEMENT type that is passed with just "ttt..."

Reply:
I don't see why we would have to rename those, but it seems like you have a reason for this one